res_syntax/src/reactjs_jsx_v3.ml

```diff
@@ -446,8 +446,7 @@ let jsxMapper ~config =
       args
   in
 
-  let rec recursivelyTransformNamedArgsForMake mapper expr args newtypes =
-    let expr = mapper.expr mapper expr in
```

Comment:
This would apply the entire JSX transform again, to `expr`, every time this function is entered.
I guess at best this would do nothing as the transform has happened already.
If it does something, it's probably not going to be something good.
Am I missing something @mununki ?

Reply:
I'm not sure why I didn't realize it didn't need this.